nss-day-cohort-39/tabloid-full-stack-bobs
# Created a form to allow users to Add New Posts

**CHANGES**
- Created a PostForm.js module
- Added the addPost function in the PostProvider.js module
- Refactored the useEffect in PostList.js
- Added a "Add Post" button in PostList.js

**TESTING**
- Fetch branch cc-create-post
- Execute the code in visual studio and run npm start on the client subdirectory in the terminal
- Login to the application and click the Posts tab
- Click the Add Post button
- Fill in the form, then click the button to save the post. 
- Once returned to the postlist, either search My Posts or All Posts to ensure that the newly added post was created and is rendered
